scripts/check_msi_installation_files_regular_debug.txt

```diff
@@ -18,246 +18,5 @@ C:\Program Files\ebpf-for-windows\drivers\eBPFCore.sys
 C:\Program Files\ebpf-for-windows\drivers\NetEbpfExt.inf
```

Comment:
Just checking that the txt files listing dependencies for other flavors don't need changing too

Reply:
Confirmed, no other changes are required (the CICD would otherwise fail, as the txt files are actively used).